src/HealthcareApis/HealthcareApis/help/Set-AzHealthcareApisService.md

```diff
@@ -0,0 +1,369 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.HealthcareApis.dll-Help.xml
+Module Name: Az.HealthcareApis
+online version: https://docs.microsoft.com/en-us/powershell/module/az.healthcareapisservice/set-azhealthcareapisservice
+schema: 2.0.0
+---
+
+# Set-AzHealthcareApisService
+
+## SYNOPSIS
+Updates an existing healthcareApis fhir service.
+
+## SYNTAX
+
+### ServiceNameParameterSet (Default)
+```
+Set-AzHealthcareApisService -Name <String> -ResourceGroupName <String> [-CosmosOfferThroughput <Int32>]
+ [-Authority <String>] [-Audience <String>] [-EnableSmartProxy] [-CorsOrigin <String[]>]
+ [-CorsHeader <String[]>] [-CorsMethod <String[]>] [-CorsMaxAge <Int32>] [-AllowCorsCredential]
+ [-AccessPolicyObjectId <String[]>] [-Tag <Hashtable>] [-AsJob] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ResourceIdParameterSet
+```
+Set-AzHealthcareApisService [-CosmosOfferThroughput <Int32>] [-Authority <String>] [-Audience <String>]
+ [-EnableSmartProxy] [-CorsOrigin <String[]>] [-CorsHeader <String[]>] [-CorsMethod <String[]>]
+ [-CorsMaxAge <Int32>] [-AllowCorsCredential] [-AccessPolicyObjectId <String[]>] [-Tag <Hashtable>]
+ [-ResourceId] <String> [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### InputObjectParameterSet
+```
+Set-AzHealthcareApisService [-Tag <Hashtable>] [-InputObject <PSHealthcareApisService>] [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Updates an existing healthcareApis fhir service.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Set-AzHealthcareApisFhirService -Name MyService -ResourceGroupName MyResourceGroup -CosmosOfferThroughput 500
+
+ResourceGroupName Name 
+----------------- -----------
+MyResourceGroup   MyService
+```
+
+Updates the existing healthcareapis service named MyService in the resource group MyResourceGroup  with the cosmosdb OfferThroughput = 500.
+
+### Example 2
+```powershell
+PS C:\> Set-AzHealthcareApisFhirService -ResourceId MyResourceID  -CosmosOfferThroughput 500
```

Comment:
@poadhika same comment about making sure a variable representing the resource ID is passed to the `-ResourceId` parameter